app/retail/emails.py

```diff
@@ -525,31 +525,30 @@ def render_start_work_applicant_expired(interest, bounty):
 def render_new_bounty_roundup(to_email):
     from dashboard.models import Bounty
     from external_bounties.models import ExternalBounty
-    subject = "A Maintainers Guide To Hacktoberfest | ETH Travels"
+    subject = "CodeFund Grows | Hacktober Begins"
 
     intro = '''
 
 <p>
 Hi there,
 </p>
 <p>
-<a href="https://medium.com/gitcoin/gitcoins-hacktoberfest-98825f199af2">Gitcoin's Hacktoberfest</a> starts on Monday! Post your favorite 3 open source projects on twitter & tag us (<a href="https://twitter.com/GetGitcoin">@GetGitcoin</a>), then make a <a href="https://gitcoin.co/requests">Gitcoin Request</a> to give those projects a boost!
+<a href="https://codefund.io/">CodeFund</a> wants to support open source projects everywhere!
+By placing ethical ads in beautiful places, Eric Berry and team have been providing Discourse, CodeSandbox, and Material-UI
```

Comment:
`<a href="https://twitter.com/codeberry"> Eric Berry</a>`

Reply:
coderberry not codeberry -- my bad

Reply:
I didn't catch this either...  Sorry about that!